CHANGES.md

```diff
@@ -23,6 +23,17 @@ OpenSSL 3.0
 
 ### Changes between 1.1.1 and 3.0 [xx XXX xxxx]
 
+ * Deprecated EVP_PKEY_set_alias_type().  Because SM2 EVP_PKEYs had unusual
+   properties and weren't recognised as such, but as usual ECC keys, this
+   function was previously needed as a workaround to switch on SM2
+   functionality.  With OpenSSL 3.0, this key type is internally recognised
+   as such, and the workaround is no longer needed.
+   
+   Functionality is still retain as it is, but will only work with EVP_PKEYs
+   with a legacy internal key.
+   
+   *Richard Levitte*
+
```

Comment:
This is fairly difficult to read.
What about simplifying it to..
``` 
* Deprecated EVP_PKEY_set_alias_type().  This function was previously needed as a workaround for SM2. With OpenSSL 3.0, this key type is internally recognized
so the workaround is no longer needed.
Functionality is still retain as it is, but will only work with EVP_PKEYs with a legacy internal key.

```

Reply:
Sure

Reply:
Done